### PR TITLE
Homepage redo

### PR DESCRIPTION
**Pull request by ankit1khare**
Notes:
1. Products growth kit could be designed as buttons with icons.
2. Under Help, the first bullet, docs, will be replaced with tutorials when we move to the new tool
3. Statsig is the complete platform for product growth, empowering your team to build and innovate like the best tech companies in the world - we can add something like Don't take our word for it, and then a button right after this saying something like, "Try Yourself".
- Overview page redo
- SDK overview page revamp
- AI/ML folder rename

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://67051db72cd8e804a5094e9f--cozy-fox-0defec.netlify.app

File: docs/getting-started.mdx [hunk 1]
```diff
@@ -8,28 +8,59 @@ import ImportantConcepts from '../src/components/getting-started/ImportantConcep
...
 
-<ImportantConcepts />
+- **[Feature Flags](/feature-flags/working-with)**: Easily manage feature rollouts, run safe deployments, and control feature access in real-time.
+- **[Experimentation](/experiments-plus)**: Set up and run controlled experiments (A/B/n, multivariate) to measure the impact of product changes.
+- **[Product Analytics](/product-analytics/overview)**: Track key metrics that drive your product and business success.
+- **[Web Analytics](/webanalytics/overview)**: Understand how users interact with your web properties, track traffic, and analyze performance.
+- **[Session Replays](/session-replay/overview)**: Watch replays of user sessions to see exactly how users interact with your product and identify opportunities to improve.
+- **[AI/ML Optimization](/ai-ml/ai-prompt-experiment)**: Automatically optimize experiments and track the performance of AI prompts with Statsig’s AI capabilities.
+- **[Statsig Warehouse Native](/statsig-warehouse-native/introduction)**: Integrate directly with your data warehouse to run experiments on top of your existing datasets.
```
Comment (tore-statsig) [inline docs/getting-started.mdx:23]: I thought this was going to be a visual like https://latest.docs-v2.statsig.com/

that would tighten it up a bit and bring the getting started up higher

File: docs/sdks/getting-started.md [hunk 2]
```diff
@@ -4,29 +4,42 @@ sidebar_label: Getting Started
...
+  
+2. **Logging Events**: Log events for user actions or system behaviors to analyze feature performance or experiment outcomes. These events are used to compute metrics, which are accessible through the **Statsig Console** for in-depth analysis.
 
-### Client SDKs
-- [Javascript](/client/javascript-sdk)
+For web platforms, such as JavaScript and React, Statsig also supports **[autocaptured events for web analytics](/webanalytics/overview)** and **[session replay](/session-replay/overview)** to give you insights into user behavior without manual event tracking.
+
+To learn more about the differences between **client-side** and **server-side** SDKs, you can refer to our [Client vs Server SDK Overview](/sdks/client-vs-server).
+
+---
```
Comment (tore-statsig) [inline docs/sdks/getting-started.md:19]: nextjs is one of the most popular - its both client and server, so it doesnt really fit this framing.  we need to make sure it is listed prominently

File: docs/getting-started.mdx (hunk 1 above)
Comment (ankit1khare) [reply docs/getting-started.mdx:23]: agreed. The comments I have in the PR description includes this. Let me look into a way if I could quickly bring in the components from v2

**Pull request merged**